src/Nncase.Core/Quantization/QuantizeOptions.cs

```diff
@@ -30,6 +30,64 @@ public enum ModelQuantMode : int
     UseQAT,
 }
 
+/// <summary>
+/// Quant Type.
+/// </summary>
+public enum QuantType : int
+{
+    /// <summary>
+    /// uint8.
+    /// </summary>
+    Uint8,
+
+    /// <summary>
+    /// int8.
+    /// </summary>
+    Int8,
+
+    /// <summary>
+    /// int16.
+    /// </summary>
+    Int16,
+}
+
+/// <summary>
+/// Fine Tune Weights Method.
+/// </summary>
+public enum FineTuneWeightsMethod : int
+{
+    /// <summary>
+    /// no fine tune weights.
+    /// </summary>
+    NoFineTuneWeights,
+
+    /// <summary>
+    /// use sqaunt.
+    /// </summary>
+    UseSquant,
+
+    /// <summary>
+    /// use adaround.
+    /// </summary>
+    UseAdaRound,
+}
+
+/// <summary>
+/// Is Use Mix Quant Or Not.
+/// </summary>
+public enum IsUseMixQuant : int
+{
+    /// <summary>
+    /// use mix quant.
+    /// </summary>
+    UseMixQuant,
+
+    /// <summary>
+    /// not use mix quant.
+    /// </summary>
+    NotUseMixQuant,
```

Comment:
The enum name `IsUseMixQuant` sounds like a bool variable. I think when this enum will not extend its definition in the future, we can use bool var to replace it, otherwise, it needs to change the name like `MixQuantMode`